examples/cluster/plot_digits_linkage.py

```diff
@@ -2,35 +2,28 @@
 =============================================================================
 Various Agglomerative Clustering on a 2D embedding of digits
 =============================================================================
-
```

Comment:
Please revert all the changes in the first docstring. They are not linked with the purpose of the PR and this is fine to have paragraph.

Reply:
Happy to merge once there are reverted